editor/components/block-settings-menu/block-transformations.js

```diff
@@ -27,9 +27,12 @@ function BlockTransformations( { blocks, small = false, onTransform, onClick = n
 	return (
 		<Fragment>
 			<div className="editor-block-settings-menu__separator" />
+			<span
+				className="editor-block-switcher__menu-title"
+			>
+				{ __( 'Transform into:' ) }
```

Comment:
Should we preserve the translators comment?

Reply:
I felt like it was unnecessary given the label was changed from sprintf( __( 'Turn into %s' ), title ) into __( 'Transform into:' ) — and also because I can't show the translator comment in context of the label because it's no longer a variable that gets inserted.